src/main/java/no/nav/familie/integrasjoner/aktør/AktørController.kt

```diff
@@ -19,22 +18,6 @@ import org.springframework.web.bind.annotation.RestController
 class AktørController(
     private val aktørService: AktørService,
 ) {
-    /**
-     * Dette endepunktet brukes av familie-ef-iverksett for å sende fattet vedtak til Arena.
-     * Kan fjernes hvis behovet for å sende med aktørID i vedtakshendelsen forsvinner.
-     */
-    @PostMapping("v2/{tema}")
-    fun finnAktørIdForPersonIdent(
-        @RequestBody(required = true) ident: Ident,
-        @PathVariable tema: Tema,
-    ): ResponseEntity<Ressurs<Map<String, String>>> =
-        ResponseEntity.ok(
-            success(
-                mapOf("aktørId" to aktørService.getAktørIdFraPdl(ident.ident, tema)),
```

Comment:
Kan vel fjerne servicen her også ? Samt en del tester?

Reply:
Samt tilsvarende kode for client

Reply:
Men kan vi nå fjerne den andre ? Bruker ikke mottak denne fortsatt ? Ref "kan fjernes når eller hvis det blir mulig å assosiere journalposter med kun personident".